lm_eval/tasks/yanolja/ai_hub_book_summarization.yaml

```diff
@@ -2,5 +2,5 @@ task: aihub_book_ko
 dataset_name: aihub_book_ko
 dataset_path: json
 dataset_kwargs:
-  data_files: ./data/ai_hub_book_summarization.jsonl
+  data_files: puree://e0PKY7Y9zP7TOoiFApRD
```

Comment:
the URI need to be put on `dataset_path` instead of `data_files`

Reply:
I mean it should be like this:
```
dataset_name: aihub_book_ko
dataset_path: puree://e0PKY7Y9zP7TOoiFApRD
include: summarization_template.yaml
```

Please also try running one of the test to make sure that there was no another issue.